config/jobs/kubernetes/sig-release/kubernetes-builds.yaml

```
@@ -164,3 +164,33 @@ periodics:
     testgrid-alert-email: release-managers+alerts@kubernetes.io
     testgrid-dashboards: sig-release-releng-informing
     testgrid-tab-name: build-packages-rpms
+
+- interval: 6h
+  name: ci-fast-forward
+  cluster: k8s-infra-prow-build
+  decorate: true
+  spec:
+    containers:
+    - image: gcr.io/k8s-staging-releng/k8s-ci-builder:latest-default
+      imagePullPolicy: Always
+      command:
+      - wrapper.sh
+      - /krel
+      - fast-forward
+      - --non-interactive
+      - --submit
+      resources:
+        requests:
+          cpu: 4
+          memory: "8Gi"
+        limits:
+          cpu: 4
+          memory: "8Gi"
+  rerun_auth_config:
+    github_team_slugs:
+      - org: kubernetes
+        slug: release-managers
+  annotations:
+    testgrid-alert-email: release-managers+alerts@kubernetes.io
+    testgrid-dashboards: sig-release-releng-informing
+    testgrid-tab-name: fast-forward
```

Possible to have something explicit like:
```suggestion
    testgrid-tab-name: git-repo-kubernetes-fast-forward
```

---

Fixed as suggested :+1: